pytuya/__init__.py

```diff
@@ -252,8 +250,8 @@ def generate_payload(self, command, data=None):
         #print('postfix_payload %r' % hex(len(postfix_payload)))
         assert len(postfix_payload) <= 0xff
         postfix_payload_hex_len = '%x' % len(postfix_payload)  # TODO this assumes a single byte 0-255 (0x00-0xff)
-        buffer = hex2bin( payload_dict[self.dev_type]['prefix'] + 
-                          payload_dict[self.dev_type][command]['hexByte'] + 
+        buffer = hex2bin( payload_dict[self.dev_type]['prefix'] +
```

Comment:
removing trailing space is great - ideally should be a separate commit. Not the end of the world, will likely accept :)

Reply:
Yes sorry, I noticed this afterwards as well. Must be my vim config that strips trailing spaces automagically.